assets/js/src/index.js

```diff
@@ -1,6 +1,12 @@
+import { eventData } from './config.js';
+import { trackClassicIntegration } from './integrations/classic'
+
 // Initialize tracking for classic WooCommerce pages
-import { trackClassicIntegration } from './integrations/classic';
-trackClassicIntegration();
+document.addEventListener( 'DOMContentLoaded', () => {
```

Comment:
To be on the safe side we'd need to cater for the case when the event already loaded https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/client/legacy/js/frontend/order-attribution.js#L123-L127 this way we can include the script before and after and it will do the job.